lib/scripting/event-transformer.spec.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * VPDB - Virtual Pinball Database
+ * Copyright (C) 2019 freezy <freezy@vpdb.io>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import * as chai from 'chai';
+import { expect } from 'chai';
+import { ThreeHelper } from '../../test/three.helper';
+import { NodeBinaryReader } from '../io/binary-reader.node';
+import { Table } from '../vpt/table/table';
+import { EventTransformer } from './event-transformer';
+import { astToVbs, vbsToAst } from '../../test/script.helper';
+
+chai.use(require('sinon-chai'));
+
+/* tslint:disable:no-unused-expression */
+describe('The scripting event transformer', () => {
+
+	const three = new ThreeHelper();
+	let table: Table;
+
+	before(async () => {
+		table = await Table.load(new NodeBinaryReader(three.fixturePath('table-gate.vpx')));
+	});
+
+	it('should transform a valid event on a valid item', () => {
+
+		const vbs = `Sub WireRectangle_Init()\n    BallRelease.CreateBall\nEnd Sub\n`;

```

Comment:
With the upcoming changes to the grammar, we are removing the whitespace checks from the beginning of block statements. I think these tests will fail.